pom.xml

```diff
@@ -76,14 +77,14 @@
             <name>Sonatype Nexus Snapshots</name>
             <url>${sonatypeOssDistMgmtSnapshotsUrl}</url>
         </snapshotRepository>
-        <repository>
+   		<repository>
```

Comment:
Mixed intendation: opening and closing tag `<repository>` are partially intended by tabs instead spaces.